sources/functions/calibre

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+## Unused, left for another day
```

Comment:
What is the status of this entire file? I would prefer not to have whole swaths of functions which are pulled in but not used anywhere.

Reply:
Unfortunately I was no able to compile calibre from source for arm :( the apt versions are old af. Unless you know how we could do it, let's drop it and see if someone comes asking for it.